### DISCUSSION TITLE
Display "notes" in Programs column

### DISCUSSION THREAD

**Discussion by ferrlen** [tnodir/fort#362] [Ideas]
The `Programs` window has no column for a program's notes, even though one can still find the notes contents by using the search function (i.e., if I wrote in appA's notes "appA is awesome!", by searching "awesome" appA will be caught as well as MyAwesomeApp).

Would it be possible to add a column for notes? It'd make it easier for me to readily see and remember that some apps/services are important for what I'm trying to do

**Comment by tnodir** [comment]
> Would it be possible to add a column for notes?

It'll be implemented as part of #205, but not soon.

> It'd make it easier for me to readily see

You can see the Notes by hovering the Name column as its tool-tip.
